nsxt/resource_nsxt_policy_ipsec_vpn_service.go

```diff
@@ -41,7 +43,8 @@ func resourceNsxtPolicyIPSecVpnService() *schema.Resource {
 			"description":         getDescriptionSchema(),
 			"revision":            getRevisionSchema(),
 			"tag":                 getTagsSchema(),
-			"locale_service_path": getPolicyPathSchema(true, false, "Polciy path for the locale service."),
+			"locale_service_path": getPolicyPathSchema(false, false, "Polciy path for the locale service."),
```

Comment:
We should mark this as Deprecated (same for other resources)

Reply:
done